### PR TITLE
add types: react-native-easy-upgrade

### PR DESCRIPTION
**Pull request by taoqf**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by taoqf** [comment]
@typescript-bot Great!

File: types/react-native-easy-upgrade/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Type definitions for react-native-easy-upgrade 0.0
+// Project: https://github.com/react-native-org/react-native-easy-upgrade#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```
Comment (andrewbranch) [inline types/react-native-easy-upgrade/index.d.ts]: Are the few instances of `unknown` here really worth restricting the version to 3.0 and higher? Callback return types are very frequently ignored, so `void` or `any` is common. `any` also seems appropriate for the `lookupInfo` dictionary. If you did that, I think you could take the version all the way down to 2.1, so potentially a lot more people could use these typings.

File: types/react-native-easy-upgrade/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	downloadTitle: string;
+	downloadDescription: string;
+	downloadDestDirectory: string;
+	downloadApkName: string;
+	downloadApkEnd(path: string): unknown;
+	shouldCheckApkHasDownloaded: boolean;
+	onError(err: Error): unknown;
+}
+
+declare class RNEasyUpgrade {
```
Comment (andrewbranch) [inline types/react-native-easy-upgrade/index.d.ts]: This _appears_ to be named incorrectly, according to https://unpkg.io/react-native-easy-upgrade@0.0.9/index.js. The structure looks right but the name is `AppUpgrade`. `RNEasyUpgrade` appears to be something else, although I don‘t entirely understand what.

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Fix the name (or if I’m wrong, explain what I missed), and give the version some consideration.

**Review by andrewbranch — APPROVED** [review]

**Pull request merged**